File: URI/2702.py
C,B,P = list(map(int,input().split()))
Cr,Br,Pr = list(map(int,input().split()))

nao = 0

if C < Cr:
    nao += Cr - C
if B < Br:
    nao += Br - B
if P < Pr:
    nao += Pr - P

print(nao)

File: URI/C/2702.c
#include<stdio.h>

int C,B,P,Cr,Br,Pr,nao = 0;

int main(){
    scanf("%d %d %d",&C,&B,&P);
    scanf("%d %d %d",&Cr,&Br,&Pr);

    if(C < Cr)
        nao += Cr - C;
    if(B < Br)
        nao += Br - B;
    if(P < Pr)
        nao += Pr - P;

    printf("%d\n",nao);

    return 0;
}
